### ISSUE TITLE
Fine-tunning LR decrease issue

### ISSUE THREAD

**Issue by JunghyunKim242** [zhenngbolun/Learnbale_Bandpass_Filter#20]
In MBCNN paper, parameters in training is stated LR decrease (decreased when validation loss was lower than 0.001 dB for four consecutive epochs )

when fine-tune the model, is LR fixed or same with training implementation??
Could you let me know??

**Comment by zhenngbolun** [comment]
Yes, the only difference is that the LR is started as 1e-5

**Comment by JunghyunKim242** [comment]
非常感谢你的回答

but I have some more questions. 
1. About Epochs: 
I know that Train and fine-tunning would be finished when LR was lower than 1e-7. Usually how many epochs did you spend for train and fine-tunning??? 
2. About PSNR Performance: 
I have checked PSNR score is 44.04 in your paper. So I want to ask you how high PSNR after train (before fine-tunning).

我会等您的回复的
